src/components/RequestsTable/RequestsTable.tsx

```diff
@@ -341,7 +342,11 @@ const RequestsTable = () => {
         </Button>
         <Button
           appearance="primary"
-          disabled={!pagedItems.data?.hasNext || pagedItems.isFetching}
+          disabled={
+            !pagedItems.data ||
+            pagedItems.isFetching ||
+            pagedItems.data.length < 5
```

Comment:
This will enable a next button (and a page of blank entries will load when clicked) when there is exactly a multiple of 5 entries for the "last" page.  Suggest we use the Async Paging functions they have available.

https://pnp.github.io/pnpjs/concepts/async-paging/